Makefile

```diff
@@ -194,6 +199,7 @@ tools-vendoring:
 	push-handler \
 	test/unit \
 	test/e2e \
+	generate \
```

Comment:
gen-check is missing

Reply:
We have some others that are missing. I'll fix this one, and send a separate PR for the others.